cypress/integration/accessibility_Popover_spec.js

```diff
@@ -5,6 +5,13 @@ describe('Popover Accessibility check', () => {
   });
 
   it('Tests accessibility on the Popover page', () => {
-    cy.checkA11y();
+    cy.configureAxe({
+      rules: [
+        {
+          id: 'color-contrast',
```

Comment:
Could you add a comment about what color contrast is failing?

Reply:
DONE